webapp/components/CommentCard/CommentCard.vue

```diff
@@ -149,8 +148,9 @@ export default {
       return `#${this.anchor}` === anchor
     },
     reply() {
-      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      const message = { slug: this.comment.author.slug + ' ', id: this.comment.author.id }
       this.$emit('reply', message)
+      this.$toast.success(this.$t(`post.comment.addReply`))
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-23T14:44:25Z" title="Sunday, February 23rd 2020, 3:44:25 pm +01:00">Feb 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -149,8 +148,9 @@ export default {
       return `#${this.anchor}` === anchor
     },
     reply() {
-      const message = { slug: this.comment.author.slug, id: this.comment.author.id }
+      const message = { slug: this.comment.author.slug + ' ', id: this.comment.author.id }
       this.$emit('reply', message)
+      this.$toast.success(this.$t(`post.comment.addReply`))
```


I really dislike this solution. You get a success message even before a comment has been saved.